### PR TITLE
JDBC Tasks of Marian Zlatev  Tasks 2-3

### PR DESCRIPTION
**Pull request by NewestUser**
Separated task 2 and task 3 on its own.

### CODE REVIEW

File: src/com/clouway/jdbc/humanmanager/Human.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package com.clouway.jdbc.humanmanager;
+
+/**
+ * @author Marian Zlatev (mzlatev91@gmail.com)
+ */
+public class Human {
+
+  private final String name;
```
Comment (mgenov) [inline src/com/clouway/jdbc/humanmanager/Human.java]: could be public final and getters to be removed

File: src/com/clouway/jdbc/humanmanager/Trip.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package com.clouway.jdbc.humanmanager;
+
+
+/**
+ * @author Marian Zlatev (mzlatev91@gmail.com)
+ */
+public class Trip {
+  private int ucn;
```
Comment (mgenov) [inline src/com/clouway/jdbc/humanmanager/Trip.java]: .

File: src/com/clouway/jdbc/humanmanager/Visits.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.clouway.jdbc.humanmanager;
+
+/**
+ * @author Marian Zlatev (mzlatev91@gmail.com)
+ */
+public class Visits {
+
+  private String city;
```
Comment (mgenov) [inline src/com/clouway/jdbc/humanmanager/Visits.java]: .

Comment (NewestUser) [inline src/com/clouway/jdbc/humanmanager/Visits.java]: Be more descriptive.

Comment (mgenov) [inline src/com/clouway/jdbc/humanmanager/Visits.java]: public final and getters to be removed.

Comment (mgenov) [inline src/com/clouway/jdbc/humanmanager/Visits.java]: I told this already on many pull requests, so I type only ., because pattern is same.

Comment (NewestUser) [inline src/com/clouway/jdbc/humanmanager/Visits.java]: Note taken.

File: src/com/clouway/jdbc/humanmanager/Trip.java (hunk 2 above)
Comment (NewestUser) [inline src/com/clouway/jdbc/humanmanager/Trip.java]: Refractored fields of Trip class and removed getters.

File: src/com/clouway/jdbc/humanmanager/Human.java (hunk 1 above)
Comment (NewestUser) [inline src/com/clouway/jdbc/humanmanager/Human.java]: Refractored Human class's fields and removed getters.

Comment (mgenov) [inline src/com/clouway/jdbc/humanmanager/Human.java]: The code looks same.

File: src/com/clouway/jdbc/humanmanager/adapters/TripAdapter.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    }
+  }
+
+  @Override
+  public Trip fetchRow(ResultSet resultSet) {
+    return null;
+  }
+
+  @Override
+  public void updateRow(Connection conn, Trip oldValue, Trip newValue) {
```
Comment (mgenov) [inline src/com/clouway/jdbc/humanmanager/adapters/TripAdapter.java]: I just wanna mention that design smells, cause you are leaving empty methods.

**Comment by mgenov** [comment]
LGTM ! 
You can merge after you fix the domain object to use public final.

File: src/com/clouway/jdbc/humanmanager/adapters/TripAdapter.java (hunk 4 above)
Comment (NewestUser) [inline src/com/clouway/jdbc/humanmanager/adapters/TripAdapter.java]: Note taken.

**Pull request closed without merging**